src/routes/user.js

```diff
@@ -1,15 +1,15 @@
 import { Router } from 'express'
 import { create, getById, getAll, updateById } from '../controllers/user.js'
 import {
-  validateAuthentication,
-  validateTeacherRole
+  validateAuthentication
+  // validateTeacherRole
```

Comment:
I'd suggest just deleting this rather than commenting it out if not relevant.

Reply:
I left this one because if we want to implement the role change we are then going to need it.

Reply:
Yep but you can get it from commit history - that's what its for. It's bad practice to have chunks of commented out code.